test/io_tests/tunnel_test.py

```diff
@@ -50,10 +50,10 @@ def test_tunnel_request_received(self, send_ack_mock):
     @patch("xknx.io.Tunnel.send_ack")
     def test_tunnel_request_received_unsupported_frames(self, send_ack_mock):
         """Test Tunnel sending ACK for unsupported frames."""
-        # LDataInd APciPhysAddrRead from 0.0.1 to 0/0/0 broadcast - ETS scan for devices in programming mode
-        # <UnsupportedCEMIMessage description="APCI not supported: 0b0100000000 in CEMI: 2900b0d000010000010100" />
+        # LDataInd Unsupported Extended APCI from 0.0.1 to 0/0/0 broadcast
+        # <UnsupportedCEMIMessage description="APCI not supported: 0b1111111000 in CEMI: 2900b0d0000100000103f8" />
         # communication_channel_id: 0x02   sequence_counter: 0x4f
-        raw = bytes.fromhex("0610 0420 0015 04 02 4f 00 2900b0d000010000010100")
```

Comment:
This frame was actually received on my installation when doing a scan for devices in programming mode from ETS. 
What happens now if it was received again? Will it be put in the TelegramQueue? Should we test it?

Reply:
I changed this, because the APCI service `APciPhysAddrRead` is now actually supported. I have changed this to 'target' an APCI service that does not exist/is supported.

Reply:
We could turning to a positive test then. 
I plan to, one day, properly respond to the scans from ETS and now we are one step closer.

Reply:
Or have both a negative and positive one ;-)

Reply:
Sure!

Reply:
A positive test isn't needed, the test `test_tunnel_request_received` already does that. So I split up this test into a `test_tunnel_request_received_cemi_too_small` and `test_tunnel_request_received_apci_unsupported` (a test should test one feature).